Frends.PowerShell.RunScript/Frends.PowerShell.RunScript.Tests/UnitTests.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
+
+namespace Frends.PowerShell.RunScript.Tests
+{
+    [TestFixture]
+    public class UnitTests
+    {
+        /// <summary>
+        /// Setup initializes PowerShell ExecutionPolicy for CurrentUser as Unrestricted so the task can run running scripts.
+        /// </summary>
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            var command = "Set-ExecutionPolicy";
+            PowerShellParameter[] parameters = new PowerShellParameter[]
+            {
+                new PowerShellParameter
+                {
+                    Name = "ExecutionPolicy",
+                    Value = "Unrestricted"
+                },
+                new PowerShellParameter
+                {
+                    Name = "Scope",
+                    Value = "CurrentUser"
+                }
+            };
+            _ = PowerShell.RunCommand(command, parameters, new RunOptions());
+        }
+
+        [Test]
+        public void RunScript_ShouldRunScriptWithParameter()
+        {
+            var script = @"param([string]$testParam)
+$testParam
+write-output ""my test param: $testParam""";
+
+            var result = PowerShell.RunScript(new RunScriptInput
+            {
+                Parameters = new[] { new PowerShellParameter { Name = "testParam", Value = "my test param" } },
+                ReadFromFile = false,
+                Script = script,
+                LogInformationStream = true
+            }, new RunOptions());
+
+            NUnit.Framework.Assert.That(result.Result.Count, Is.EqualTo(2));
```

Comment:
You should add the namespace NUnit.Framework with `using` statement, so you don't have to add the namespace for each class you want to use.

Reply:
Fixed